fighters/kirby/src/opff.rs

```diff
@@ -55,6 +55,24 @@ unsafe fn inhale_forced_end(fighter: &mut L2CFighterCommon) {
     }
 }
 
+unsafe fn stone_escape_handler(fighter: &mut L2CFighterCommon) {
+    // disables stone if broken out of for any reason
+    if fighter.is_prev_status(*FIGHTER_KIRBY_STATUS_KIND_STONE_STONE)
+    && !VarModule::is_flag(fighter.battle_object, vars::kirby::instance::DISABLE_STONE) {
+        VarModule::on_flag(fighter.battle_object, vars::kirby::instance::DISABLE_STONE);
+    }
+
+    // enables movement on frame 14
+    if fighter.is_status(*FIGHTER_KIRBY_STATUS_KIND_STONE_END)
+    && MotionModule::frame(fighter.module_accessor) >= 14.0 {
+        if fighter.is_situation(*SITUATION_KIND_AIR) {
+            if KineticModule::get_kinetic_type(fighter.module_accessor) != *FIGHTER_KINETIC_TYPE_FALL {
+                KineticModule::change_kinetic(fighter.module_accessor, *FIGHTER_KINETIC_TYPE_FALL);
+            }
+        }
+    }
```

Comment:
Kinetic types don't need to be constantly set/changed, just once. This can be placed in the `game` acmd for `special_lw2` and `special_air_lw2` on frame 14, and also should not need to check the kinetic type beforehand